azure_media_services/templates/player.html

```diff
@@ -28,15 +28,16 @@
     </div>
 % endif
 
-<ul class="wrapper-downloads">
+<div class="wrapper-downloads azure-media-wrraper-download-container">
 % if download_url:
-	<li class="video-sources video-download-button">
-		<a href="${download_url}">Download video</a>
-	</li>
-% endif	
+	<div class="video-sources transcript-download-btn-style">
```

Comment:
another nit: indenting prior to this line uses 4 spaces, but we start using tabs on line 33. please normalize to spaces from line 33 and below.